### PR TITLE
Searchbarint new

### PR DESCRIPTION
**Pull request by Jchuma98**
Screenshot 2022-03-05 at 17 36 10

### CODE REVIEW

File: app/models/flat.rb [hunk 1]
```diff
@@ -2,28 +2,21 @@ class Flat < ApplicationRecord
...
   has_many_attached :photos
-
   validates :address, presence: true
   validates :price, presence: true
   validates :description, presence: true, length: { minimum: 10 }
   validates :city, presence: true
-
-  validates :photos, presence: true
-
+  #validates :photos, presence: true
```
Comment (caziliot) [inline app/models/flat.rb:14]: Why did you commented this line?

Comment (caziliot) [inline app/models/flat.rb:14]: Why did you commented this line?

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -26,12 +26,17 @@ import "bootstrap";
...
 // Internal imports, e.g:
 import { initFlatpickr } from "../plugins/flatpickr";
+import { initFlatRange } from "../plugins/flatpickr";
 import { initChatRoomCable } from "../channels/chat_room_channel";
 import { initAutocomplete } from "../plugins/init_autocomplete";
 
 document.addEventListener('turbolinks:load', () => {
   // Call your functions here, e.g:
-  initFlatpickr();
+  initFlatpickr("#month_start", {
```
Comment (caziliot) [inline app/javascript/packs/application.js:35]: Remember not to put this methods in the application.js, makes it a little messy.

Comment (caziliot) [inline app/javascript/packs/application.js:35]: Remember not to put this methods in the application.js, makes it a little messy.

**Comment by caziliot** [comment]
Joaquin, I have not merged your pull request, I enter the branch, and there are a lot of stuff that aint working from design and is bit behind, though I took some of the features of the search logic, still the start month and end month is to be implemented... as the available month. 



I think is best not to merge the pull requests, because is harder to find the mistakes, I would ask the team so they decide. 



On the other hand, we agree you would work on the chat bubbles... is totally cool if you don't like that, but it just makes it really difficult if you don't let us know what you would be working on, because then this things happen. We are to help each other, is all cool for next time use the trello cards or the stack channel to discuss tasks.

**Comment by caziliot** [comment]
Joaquin, I have not merged your pull request, I enter the branch, and there are a lot of stuff that aint working from design and is bit behind, though I took some of the features of the search logic, still the start month and end month is to be implemented... as the available month. 



I think is best not to merge the pull requests, because is harder to find the mistakes, I would ask the team so they decide. 



On the other hand, we agree you would work on the chat bubbles... is totally cool if you don't like that, but it just makes it really difficult if you don't let us know what you would be working on, because then this things happen. We are to help each other, is all cool for next time use the trello cards or the stack channel to discuss tasks.

File: app/models/flat.rb (hunk 1 above)
Comment (Jchuma98) [reply app/models/flat.rb:14]: because it already there

**Comment by Jchuma98** [comment]
Screenshot 2022-03-10 at 23 06 53
 new search bar bug fix layout

**Pull request merged**